src/FormElement/ScheduleElement.php

```diff
@@ -495,6 +496,43 @@ protected function assemble()
                     'valid'     => function (): bool {
                         return $this->isValid();
                     },
+                    'validators' => [
```

Comment:
I don't quite like the usage of a validator here. You're essentially mis-using it to define the element's content.

Can you use the `onRegister()` method of the element? This way you can remember the schedule element as parent and also replace the `valid` callback.

The parent is then used to decide what to do in `assemble()`.